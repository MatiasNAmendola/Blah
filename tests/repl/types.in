
15
True
False
'This is a string?'
'\'\\\'\\'
'New\nLine\fWord\bld\r'
'This continues
and \o\w\o\i\e\s'
'\047\x5F\x7d\u300e„Äè\256
\248\u314G'
[]
['some\n\t\\\'\vth ings\b\xa9\nare\fgood\0\r']
[True and '',
113, 23 -
-12 * -2
,51!=84]
[	[
],      True,['Nested']
,[1,       2,'[]',
4,5]]
[2 * (
25 -
15
), 12]
Nothing
