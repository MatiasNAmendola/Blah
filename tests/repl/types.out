15
True
False
This is a string?
'\'\
New
LineWordld
This continues
and \o\w\o\i\e\s
'_}『』®
\248\u314G
[]
['some\n\t\\\'\vth ings\b©\nare\fgood\0\r']
['', 113, -1, True]
[[], True, ['Nested'], [1, 2, '[]', 4, 5]]
[20, 12]
Nothing
